src/client/apps/JudgeApp.ts

```diff
@@ -85,6 +85,8 @@ export class JudgeApp extends Application<JudgeAppState> {
   }
 
   loadAssignment() {
+    const start = new Date().getTime();
```

Comment:
Now that I realize it There may be a chance that someone looks at a previous assignment the rated to see the score they gave and this would reset the timer I think. Or if they go back and decide to change a score. The logic we have is probably fine for now but you should open a ticket to investigate/resolve these issues. You can work on it before or after the hackathon depending on what you want to prioritize, but the data may not be that clean

Reply:
I think the only time this issue will be applicable if they actually change the score because that's when submitassignment is called which has the computation of elapsed time. Which is what gets stored in the table. But I agree that it's something to look at.